QATester/com/generic/QATester/QATester.java

```diff
@@ -0,0 +1,176 @@
+package com.generic.QATester;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import com.generic.model.Warehouse;
+import com.generic.tracker.WarehouseTracker;
+
+
+import com.generic.model.FreightType;
+import com.generic.model.Shipment;
+
+
+/**
+ * A Junit class for unit testing
+ * @author Seyi Ola
+ *
+ */
+
+public class QATester {
+	
```

Comment:
For example, if you want to test Warehouse class,
by convention, you create a file called WarehouseTest.java
and write it's test inside of said file.

and if you want to test Shipment class, you would create a file called ShipmentTest.

This creates modular tests that can be executed as a standalone

Reply:
Noted